SPECS/xapi.spec

```diff
@@ -1375,6 +1404,268 @@ Coverage files from unit tests
 %{?_cov_results_package}
 
 %changelog
+* Tue Apr 15 2025 Gaëtan Lehmann <gaetan.lehmann@vates.tech> - 25.6.0-1.1
+- Upstream changelog:
```

Comment:
We list removed, rediffed and added patches in here, as this is relevant packaging information. See Yann's changelog entry from Feb 14.